test/TestProjects/SingleTopLevelClientWithOperations-LowLevel/Generated/Client4.cs

```diff
@@ -27,6 +27,9 @@ public partial class Client4
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
         public virtual HttpPipeline Pipeline => _pipeline;
 
+        /// <summary> The HTTP Uri. </summary>
+        public virtual Uri Uri => _endpoint;
```

Comment:
This is interesting.  Should [subclients/resource subclients](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-subclients) have `Uri` on them -- will this always be available?

Reply:
@pshao25, how do we decide what the value of `_endpoint` will be for a subclient?

Reply:
Are you saying subclients should not have Uri? 
The value of _endpoint is passed from service client when the sub client is created.

Reply:
If there is no endpoint, there would be no property at all. The origin of the issue is that property is required for logging purposes. In this perspective, I don't see the difference between client and subclient.

Reply:
@pshao25, here is some additional context: https://github.com/Azure/autorest.csharp/pull/1905#issuecomment-1036455832

Given this, we will not put a `Uri` property on subclients.  We will, however, put an `Endpoint` property on subclients, if they have an endpoint that is passed in to the subclient constructor.